Dark.json

```diff
@@ -3725,6 +3726,7 @@
         "gametext": "May carry 2 passengers. Bantha ability = 1/2. May move as a 'react' only to a battle or Force drain (if within range). Each Tusken Raider at same exterior site is power +1.",
         "lore": "Transport, pack animal. Many breeds of different sizes and colors. Three meters tall. Can go weeks without food or water. Found throughout the galaxy."
       },
+      "rules": [{"name":"Creature Rules", "url":"https://www.starwarsccg.org/creature-rules/"}],
```

Comment:
Greg noticed this on Slack. We need to remove these "Creature Rules" entries for Creature Vehicles (Only including them for actual creatures).